src/v2/params/types.ts

```diff
@@ -28,16 +28,13 @@ export interface ParamSpec<T = unknown> {
   default?: T;
   label?: string;
   description?: string;
-  valueType?: ParamValueType;
+  type: ParamValueType;
```

Comment:
nit* If we are going to change the name, should we also change the name of `type ParamValueType` to `type ParamType`?